build/external-dependencies.props

```diff
@@ -237,6 +237,7 @@
     <ExternalDependency Include="Microsoft.ApplicationInsights.AspNetCore" Version="$(MicrosoftApplicationInsightsAspNetCorePackageVersion)" Source="$(DefaultNuGetFeed)" />
     <ExternalDependency Include="Microsoft.AspNet.Identity.EntityFramework" Version="$(MicrosoftAspNetIdentityEntityFrameworkPackageVersion)" Source="$(DefaultNuGetFeed)" />
     <ExternalDependency Include="Microsoft.AspNet.WebApi.Client" Version="$(MicrosoftAspNetWebApiClientPackageVersion)" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="Microsoft.Azure.DocumentDB.Core" Version="$(MicrosoftAzureDocumentDBCorePackageVersion)" Source="$(DefaultNuGetFeed)" />
```

Comment:
cc: @Eilon  for new external dependency.

Reply:
Talked in person - no OSS approval needed because this is MSFT. However, it is a non-PCP dependency, but as of right now this isn't used in a "shipping" part of the product, so no concern.